src/config/frontend/EdFi.DmsConfigurationService.Frontend.AspNetCore.Tests.Unit/Modules/IdentityModuleTests.cs

```diff
@@ -187,7 +187,7 @@ public async Task When_provider_has_bad_credentials()
         string content = await response.Content.ReadAsStringAsync();
 
         // Assert
-        response.StatusCode.Should().Be(HttpStatusCode.Unauthorized);
+        response.StatusCode.Should().Be(HttpStatusCode.BadGateway);
```

Comment:
According to DMS-389 we should not be simply forwarding IdP error status codes to the client. Our status code should be Bad Gateway with enough information in the error to determine why.